io.github.multiscrobbler.metainfo.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="console-application">
+    <id>io.github.multiscrobbler</id>
```

Comment:
the app-id is incorrect. It should be `io.github.`user`.`application. (ie have 4 components)